### PR TITLE
Use newer S3 node package to patch security vulnerability with mime < 1.4.1

### LINKED ISSUE #9: Security vulnerability in S3 node package dependency
@diracdeltas mentioned that there's a security vulnerability in a dependency (`mime < 1.4.1`) used by brave/brave-core-crx-packager. Investigation turned up that this is due to a dependency on a (now abandoned) S3 module in brave/release-tools. That S3 module has been forked with updated dependencies (see https://github.com/andrewrk/node-s3-client/issues/190), so we should update release-tools to use that newer S3 module.

### PR DESCRIPTION
**Pull request by emerick**
Fixes #9 
---
I tested by running the following command locally, which uses release-tools to upload the Brave Dark Theme extension to S3:

```
node bin/updateExtensions.js \
     --chromium=0.0.0.0 \
     --id bfdgpgibhagkpdlnjonhkabjoijopoge \
     --location=../vault-updater/data \
     --path=c:/mongoose/bfdgpgibhagkpdlnjonhkabjoijopoge/extension_1_0_0.crx \
     --version=1.0.0 \
     --v=2
```

### CODE REVIEW

**Comment by emerick** [comment]
@aekeus / @bsclifton This is to address a security vulnerability in `mime 1.4.1`, see #9 for additional details. I don't have permission to add a reviewer to this it seems, but was hoping one of you could take a look.

**Review by bsclifton — APPROVED** [review]
LGTM - once merged, we can easily test by doing browser-laptop builds

**Comment by emerick** [comment]
Thanks @bsclifton!

**Review by aekeus — APPROVED** [review]

**Pull request merged**
